docs/platform/api/create-block-storage-volumes-with-the-linode-api.md

```diff
@@ -0,0 +1,197 @@
+---
+author:
+  name: Jared Kobos
+  email: docs@linode.com
+description: Linode’s Block Storage service allows you to attach additional storage volumes to your Linode. This guide demonstrates how to create, attach, clone, and resize Volumes using the Linode API.
+og_description: Linode’s Block Storage service allows you to attach additional storage volumes to your Linode. This guide demonstrates how to create, attach, clone, and resize Volumes using the Linode API.
+keywords: ["linode api", "block storage", "volume"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2018-05-03
+modified_by:
+  name: Linode
+published: 2018-05-03
+title: Block Storage Volumes with the Linode API
+external_resources:
+  - '[API Documentation](https://developers.linode.com/v4/introduction)'
+---
+
+<!--- ![Block Storage Volues with the Linode API](/docs/assets/api-block-storage/api-block-storage-smp.png) --->
+
+The Linode API allows you to create, delete, attach/detach, clone, and resize Block Storage Volumes.
+
+## Before You Begin
+
+You will need a Personal Access Token to access the `/volumes` endpoint. Access tokens can be created through the beta [Linode Manager](https://cloud.linode.com). See our [Getting Started with the Linode API](/docs/platform/api/getting-started-with-the-linode-api/) for more information.
```

Comment:
If you linked [Linode Manager](#) directly to https://cloud.linode.com/profile/tokens, it might be more helpful